packages/console/src/scss/transfer.module.scss

```diff
@@ -28,6 +28,12 @@
   overflow-y: auto;
 }
 
+.emptyBoxContent {
```

Comment:
make the default boxContent css display flex? if not empty, child element flex 1?

Reply:
exactly

Reply:
Or use `height: 100%` in the child element to make sure it takes 100% height of the `boxContent` div

Reply:
Cool, updated.